src/index.js

```diff
@@ -1,6 +1,34 @@
 var Cookies = require('js-cookie');
-Cookies.removeAll = function (attributes) {
+
+var extend = function () {
+	var i = 0;
+	var result = function() {};
+	for (; i < arguments.length; i++) {
+		var argument = arguments[ i ];
+		var argumentKeys= Object.keys(argument)
+		argumentKeys.forEach(function(key) {
+			result[key] = argument[key];
+		});
+	}
+	return result;
+};
+
+var removeAll = function (attributes) {
 	Object.keys(Cookies.get()).forEach(function (cookie) {
 		Cookies.remove(cookie, attributes);
 	});
 };
+
+var decorate = function (api) {
+	return extend(api, {
+		withConverter: function() {
+			var apiWithConverter = Cookies.withConverter.apply(this, arguments);
+			return decorate(apiWithConverter);
+		},
+		removeAll: removeAll
+	});
+};
+
+Cookies.removeAll = removeAll;
+
+module.exports = decorate(Cookies);
```

Comment:
Here I am using the default node module pattern (`module.exports`) instead of just changing the cached api from `js-cookie` module.